app/controllers/stories/tagged_articles_controller.rb

```diff
@@ -69,7 +69,7 @@ def stories_by_timeframe(stories:)
       elsif params[:timeframe] == Timeframe::LATEST_TIMEFRAME
         stories.where(score: -20..).order(published_at: :desc)
       else
-        stories.order(hotness_score: :desc).where(score: Settings::UserExperience.home_feed_minimum_score..)
+        stories.order(hotness_score: :desc).with_at_least_home_feed_minimum_score
```

Comment:
Are we ok with implicitly adding `.featured` here?

Reply:
Given that as of two weeks ago no articles in DEV.to had had featured since June 2021, I'm alright with adding that.

The `articles.featured` attribute is currently under consideration for its future.